src/schema/v2/artwork_version.ts

```diff
@@ -70,3 +71,5 @@ export const ArtworkVersionResolver: GraphQLFieldConfig<
       ? authenticatedArtworkVersionLoader(id)
       : null,
 }
+
+export default ArtworkVersionResolver
```

Comment:
There's some magic inference going on in the [Node type](https://github.com/artsy/metaphysics/blob/b79c3e635927ebf8aa57b7ea6e687b11c52fa0bb/src/schema/v2/object_identification.ts#L143-L148) that makes it easier to have a default export for the field.

Reply:
whoa wait I must be losing my marbles - I _swear_ @zephraph and I had added the default export AND the node interface. no clue what happened there, but thanks for catching the issue @mzikherman !